vote/src/vote_account.rs

```diff
@@ -185,15 +219,25 @@ impl VoteAccounts {
         if stake == 0u64 {
             return;
         }
+
         let Some(staked_nodes) = self.staked_nodes.get_mut() else {
             return;
         };
-        if let Some(node_pubkey) = vote_account.node_pubkey().copied() {
+
+        VoteAccounts::do_add_node_stake(staked_nodes, stake, vote_account.node_pubkey().copied());
+    }
+
+    fn do_add_node_stake(
```

Comment:
Ditto, adding this fn and using it in `add_node_stake` would have been nice as a separate commit.

Reply:
I now put back do_add_node_stake and do_sub_node_stake in the commit that adds the entry API as I had originally done before messing up a rebase